xwiki-platform-core/xwiki-platform-webjars/xwiki-platform-webjars-api/src/main/java/org/xwiki/webjars/internal/WebJarsResourceReferenceHandler.java

```diff
@@ -120,4 +115,19 @@ protected ClassLoader getClassLoader(String namespace)
     {
         return this.classLoaderManager.getURLClassLoader(namespace, true);
     }
+
+    @Override
+    protected String getContentType(InputStream resourceStream,
+        WebJarsResourceReference resourceReference) throws IOException
+    {
+        String mimeType;
+        if (!isResourceCacheable(resourceReference) && getResourceName(resourceReference)
+            .endsWith(LESS_FILE_EXTENSION))
+        {
+            mimeType = "text/css";
```

Comment:
I would add a comment explaining why we change the content type from LESS to CSS.

Reply:
Comment added https://github.com/xwiki/xwiki-platform/pull/1586/files#diff-45ea9c87d5fb68cd5db0da7f78cf25e76f1325f5fe56e21618b21786fc706236R124-R126